tests/framework/pkg/session/session.go

```diff
@@ -10,6 +10,7 @@ type Session struct {
 	cleanupQueue []CleanupFunc
 	open         bool
 	testingT     *testing.T
+	subSession   *Session
```

Comment:
Sessions do not need to track their sub sessions like this.  When the sub session is created it registers it's cleanup function with the parent.